submissions/rubyAdeniregun/php/listview.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+require_once("db_connection.php");
+
+echo "<h5> List View Page </h5>";
+
+try{
```

Comment:
This is what looks like what you are asked to do.

But, With the way you implement this, each time I reload this page, this query `"SELECT * FROM customer_details"` will always run. Which is not what we want to do. You have to look for a way to store the result gotten from this query in a cookie. So if cookie is set then you don't want to run the query again but get the data from the cookie store